tests/traced_data/resources/coda_import_data.txt

```diff
@@ -0,0 +1,5 @@
+id;timestamp;owner;data;schemeId;schemeName;deco_codeValue;deco_codeId;deco_confidence;deco_manual;deco_timestamp;deco_author
+6469567482;;3885485969;female;1;CodaCodedGender;F;1-48;0.95;true;Fri May 18 2018 12:26:14 GMT+0100 (BST);
+4597828651;;9051968373;m;1;CodaCodedGender;M;1-01;0.95;true;Fri May 18 2018 12:26:13 GMT+0100 (BST);
+1020892817;;2494941557;WoMaN;1;CodaCodedGender;F;1-48;0.95;true;Fri May 18 2018 12:26:12 GMT+0100 (BST);
+8083248450;;7380450267;27;1;CodaCodedGender;;;;;;
```

Comment:
Why is this a .txt file, not a .csv?

Reply:
Because that's what Coda's export function gives you...

Shall I make an issue for this too?